liblangutil/EVMVersion.h

```diff
@@ -81,7 +83,7 @@ class EVMVersion:
 	bool canOverchargeGasForCall() const { return *this >= tangerineWhistle(); }
 
 private:
-	enum class Version { Homestead, TangerineWhistle, SpuriousDragon, Byzantium, Constantinople };
+	enum class Version { Homestead, TangerineWhistle, SpuriousDragon, Byzantium, Constantinople, Petersburg };
 
 	EVMVersion(Version _version): m_version(_version) {}
 
```

Comment:
The line below needs to be updated - or do you want do do that in a separate PR?

Reply:
The default version? Yes, I think that belongs in a separate one as it changes other parts of the code/tests.